_health-care/_js/_components/date_input.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import _ from 'lodash';
+
+class DateInput extends React.Component {
+  constructor() {
+    super();
+    this.state = {month: '', date: '', year: ''};
```

Comment:
yup. ES6. Double-edged sword.  React is actually kinda awesome. If you add getInitialState() to an es6 class, it yells at you in the javascript console telling you what to do instead!